site/_docs/algebra.md

```diff
@@ -37,7 +37,7 @@ planner engine generates an alternative expression that has the same
 semantics as the original but a lower cost.
 
 The planning process is extensible. You can add your own relational
-operators, planner rules, cost model, and statistics.
+operators, planner rules, cost models, and statistics.
```

Comment:
minor remark: I think the original one (in singular) was arguably more accurate, since the cost model is a single one in the whole process (as described in the previous paragraph). So, I'd lean towards leaving this sentence as it was.

Reply:
Reverted back. https://github.com/apache/calcite/pull/2245/commits/8295c062503f89eef6368c6849eaea2a73b8cf95